common/nym-lp/src/message.rs

```diff
@@ -72,6 +80,8 @@ impl MessageType {
             0x0001 => Some(MessageType::Handshake),
             0x0002 => Some(MessageType::EncryptedData),
             0x0003 => Some(MessageType::ClientHello),
+            0x0004 => Some(MessageType::KKTRequest),
```

Comment:
suggestion for this, and other variants to not duplicate the magic values:
```suggestion
            n if n == MessageType::KKTRequest as 16 => Some(MessageType::KKTRequest),
```

Reply:
Found a num_enum crate which is exactly what we need here